src/vm/analysis/contract_interface_builder/mod.rs

```diff
@@ -365,6 +371,7 @@ impl ContractInterface {
             maps: Vec::new(),
             fungible_tokens: Vec::new(),
             non_fungible_tokens: Vec::new(),
+            clarity_version: ClarityVersion::Clarity1,
```

Comment:
Why do we want to default to `ClarityVersion::Clarity1` here?

Reply:
Yep, there's no need for a default here, changed in ab8204133